woocommerce/woocommerce
# Add the "Campaigns" card into Marketing page

### All Submissions:


### Changes proposed in this Pull Request:

Addresses part of https://github.com/woocommerce/woocommerce/issues/34905.

This PR adds the new "Campaigns" card into the multichannel Marketing page.

![image](https://user-images.githubusercontent.com/417342/208299001-118aa4f2-1aaa-4784-9a07-6221e507bb3a.png)

### How to test the changes in this Pull Request:

Pre-conditions: Enable WooCommerce Multichannel Marketing experience in WooCommerce Settings > Advanced > Features page: `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`

1. Go to WooCommerce Marketing page: `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`
2. You should see the Campaigns card like the screenshot above.

### Other information:


### FOR PR REVIEWER ONLY:
